lib/remoteproc/remoteproc_virtio.c

```diff
@@ -151,17 +156,20 @@ static void rproc_virtio_reset_device(struct virtio_device *vdev)
 		rproc_virtio_set_status(vdev,
 					VIRTIO_CONFIG_STATUS_NEEDS_RESET);
 }
+#endif
 
 virtio_dispatch remoteproc_virtio_dispatch_funcs = {
 	.get_status =  rproc_virtio_get_status,
-	.set_status = rproc_virtio_set_status,
 	.get_features = rproc_virtio_get_features,
+	.read_config = rproc_virtio_read_config,
+	.notify = rproc_virtio_virtqueue_notify,
+#ifndef VIRTIO_SLAVE_ONLY
+	.set_status = rproc_virtio_set_status,
```

Comment:
Referring to Linux kernel include/uapi/linux/virtio_config.h, both slave and master can set status